docker-build/lom-slave-bullseye/lom-slave-bullseye-admin.log

```diff
@@ -0,0 +1,52 @@
+#1 [internal] load .dockerignore
```

Comment:
Are these log files only internal to slave docker ?

Reply:
yep